html-forms/index.html

```diff
@@ -28,7 +28,8 @@ <h1>Coding Form!</h1>
     <input id="javascript" type="radio" name="language" value="javascript" />
     <label for="javascript">JavaScript</label>
     <p> Would you like to sign up for more information?</p>
-    <input id="information" type="checkbox" name="information" /> Sign me up!</label>
+    <input id="information" type="checkbox" name="information"/>
+    <label>Sign me up!</label>
```

Comment:
This `label` needs a `for` attribute that matches the `id` attribute on its paired form control element.